ui/page/settings/view.jsx

```diff
@@ -211,11 +211,20 @@ class SettingsPage extends React.PureComponent<Props, State> {
             <Card
               title={__('Sync')}
               subtitle={
-                walletEncrypted && !storedPassword
-                  ? __("To enable this feature, check 'Save Password' the next time you start the app.")
-                  : null
+                walletEncrypted && !storedPassword ? (
+                  storedPassword === '' ? (
+                    <>
+                      {__('Your wallet password cannot be empty string. Sync is disabled for now.')}
```

Comment:
I don't think this is the message we should show. Why do we have to disable it for empty string?

Reply:
It would prevent an encrypted wallet from getting decrypted via a sync apply + no password  (this sets a new password). Probably should disallow this on the SDK side and force with a flag. It shouldn't happen unless someone is in a really strange scenario, so I'm okay with removing it for now.